app/controllers/apartments_controller.rb

```diff
@@ -3,4 +3,15 @@ def index
         apartments = Apartment.all
         render json: apartments
     end
+
+    def create
+        apartment = Apartment.create
+        render json: apartment
+    end
```

Comment:
I don't think this method should be on this PR as this is not part of the Trello card.

Reply:
When we ran the test for the destroy method, it failed with a message saying there was no create method. When we added the create method the test passed.